pytest_fixtures/component/leapp.py

```diff
@@ -0,0 +1,16 @@
+# LEAPP Fixtures
+import pytest
+
+
+@pytest.fixture(scope="function")
```

Comment:
since function is the default scope, you can make this implicit
```suggestion
@pytest.fixture
```